tenzir/integration/tests.yaml

```diff
@@ -1223,6 +1225,10 @@ tests:
         input: data/syslog/cef-over-syslog.log
       - command: exec 'read syslog | parse content cef'
         input: data/syslog/cef-over-syslog.log
+      - command: exec 'read syslog'
+        input: data/syslog/cef-over-syslog-without-tags.log
+      - command: exec 'read syslog | parse message cef'
```

Comment:
One final thing: Can you help me understand whether we only need `message` (as opposed to having just `content`) for this very use-case, where the `CEF` tag is needed to parse it as CEF? Or do you think that this is also convenient to have in general? The duplication bothers me a bit, in particular if the "full" message can always be reconstructed from the components. What do you think?

Reply:
`message` can't strictly be reconstructed from the constituent parts: any whitespace after the colon is lost. For a `message` `FOO: bar`, `app_name` is `FOO`, and `content` is `bar`, which is the same as for a `message` `FOO:bar`.

I feel inclined to have both `message`, and `app_name`+`process_id`+`content`, at least for now, to not accidentally lose any other information.

Reply:
Losing some initial whitespace seems fine to me, but I'll leave this up to you. You could also take a look at how other syslog parsers represent their results.

Reply:
It seems like, in general, other syslog parsers tend to not have a `message`, but `app_name`+`process_id`+`content`. I can just remove it, it shouldn't be too hard to add it back later.